internal/cmd/gentypes/main.go

```diff
@@ -385,6 +385,18 @@ import (
 				flattenAnon,
 			},
 		},
+		{
+			"LinkCreateKprobeMulti", retFd, "link_create", "BPF_LINK_CREATE",
+			[]patch{
+				chooseNth(4, 3),
+				replace(enumTypes["AttachType"], "attach_type"),
+				flattenAnon,
+				replace(pointer, "cookies"),
+				replace(pointer, "addrs"),
+				replace(pointer, "syms"),
+				rename("flags", "kprobe_multi_flags", 1),
```

Comment:
@lmb Any ideas to avoid the `skip` parameter to `rename()`?

Maybe we could add a dot operator to the first param so we could do:

```
chooseNth(4, 3), // Resolve the anon union to 'kprobe_multi' first
rename("kprobe_multi.flags", "kprobe_multi_flags") // Rename a field in the kprobe_multi child struct
flattenAnon,
```

```
struct { /* struct used by BPF_LINK_CREATE command */
  ...
  __u32		flags;		/* extra flags */
  union {
    ...
    struct {
      __u32		flags;
      __u32		cnt;
      __aligned_u64	syms;
      __aligned_u64	addrs;
      __aligned_u64	cookies;
    } kprobe_multi;
    ...
  };
} link_create;
```

Reply:
I pushed some changes that make it so we don't need the extra arg.

Reply:
Thanks!